specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/examples/createOrUpdateDeployment.json

```diff
@@ -0,0 +1,67 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "endpointName": "testendpoint",
+    "deploymentName": "testdeployment",
+    "api-version": "2020-05-01-preview",
+    "parameters": {
+      "modelUris": [
```

Comment:
modelIds (arm ids)  - also need to check if linked access check supports collection